lib/src/rules/sort_child_properties_last.dart

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/analyzer.dart';
+import 'package:linter/src/util/flutter_utils.dart';
+
+const _desc = r'Sort child properties last in widget instance creations.';
+
+const _details = r'''
+Sort child properties last in widget instance creations.  This improves
+readability and plays nicest with UI as Code visualization in IDEs with UI as
+Code Guides in editors (such as IntelliJ) where Properties in the correct order 
+appear clearly associated with the constructor call and separated from the 
+children.
+
+**BAD:**
+```
+return Scaffold(
+  appBar: AppBar(
+    title: Text(widget.title),
+  ),
+  body: Center(
+    child: Column(
+      children: <Widget>[
+        Text(
+          'You have pushed the button this many times:',
+         ),
+        Text(
+          '$_counter',
+          style: Theme.of(context).textTheme.display1,
+         ),
+      ],
+      mainAxisAlignment: MainAxisAlignment.center,
+    ),
+    floatingActionButton: FloatingActionButton(
+      child: Icon(Icons.add),
+      onPressed: _incrementCounter,
+      tooltip: 'Increment',
+    ),
+  ),
+);
+```
+
+**GOOD:**
+```
+return Scaffold(
+  appBar: AppBar(
+    title: Text(widget.title),
+  ),
+  body: Center(
+    mainAxisAlignment: MainAxisAlignment.center,
+    child: Column(
+      children: <Widget>[
+        Text(
+          'You have pushed the button this many times:',
+         ),
+        Text(
+          '$_counter',
+          style: Theme.of(context).textTheme.display1,
+         ),
+      ],
+    ),
+    floatingActionButton: FloatingActionButton(
+      onPressed: _incrementCounter,
+      tooltip: 'Increment',
+      child: Icon(Icons.add),
+    ),
+  ),
+);
+```
+''';
```

Comment:
These are not the same layouts, you moved mainAxisAlignment from inside the Column to being a property on Center. I couldn't for the life of me figure out how this lint could imply that but I'm assuming it was just a mistake.

Reply:
https://github.com/dart-lang/linter/pull/2051